src/appleseed.shaders/src/maya/as_maya_condition.osl

```diff
@@ -118,7 +118,7 @@ shader as_maya_condition
         string shadername = "";
         getattribute("shader:shadername", shadername);
 
-        warning( "[WARNING]: invalid operation in shader %s, %s:%i\n",
+        warning("[WARNING]: invalid operation in shader %s, %s:%i\n",
```

Comment:
Do we need the [WARNING] prefix here, given that we're already calling the `warning()` function?